ui/src/App.tsx

```diff
@@ -138,101 +230,150 @@ const makeOffer = () => {
-          <div style={{ textAlign: 'left' }}>
-            {istPurse && (
-              <div>
-                <b>IST: </b>
-                {stringifyAmountValue(
-                  istPurse.currentAmount,
-                  istPurse.displayInfo.assetKind,
-                  istPurse.displayInfo.decimalPlaces
+          {bldPurse && (
+            <div>
+              <b>BLD: </b>
+              {stringifyAmountValue(
+                bldPurse.currentAmount,
+                bldPurse.displayInfo.assetKind,
+                bldPurse.displayInfo.decimalPlaces
+              )}
+            </div>
+          )}
+          {invitationPurse && (
+            <div>
+              <h4>Pending Swaps:  </h4>
+              {
+                swaps.length ? (
+                  <ul style={{ marginTop: 0, textAlign: 'left' }}>
+                    {swaps.map(
+                      ({ description, customDetails }, index) => (
+                        <li key={index}>
+                          <b>{description}</b><br />
+                          {stringify(customDetails)}
+                        </li>
+                      )
+                    )}
+                  </ul>
+                ) : (
+                  'None'
                 )}
-              </div>
-            )}
-            {bldPurse && (
-              <div>
-                <b>BLD: </b>
-                {stringifyAmountValue(
-                  bldPurse.currentAmount,
-                  bldPurse.displayInfo.assetKind,
-                  bldPurse.displayInfo.decimalPlaces
+              <h4>Invitations:  </h4>
+              {
+                invites.length ? (
+                  <ul style={{ marginTop: 0, textAlign: 'left' }}>
+                    {invites.map(
+                      ({ description, customDetails }, index) => (
+                        <li key={index}>
+                          {description} {stringify(customDetails)}
+                        </li>
+                      )
+                    )}
+                  </ul>
+                ) : (
+                  'None'
                 )}
-              </div>
-            )}
-            {wallet && invitationPurse && (
-              <div>
-                <b>Invitations:  </b>
-                {
-                  invites.length ? (
-                    <ul style={{ marginTop: 0, textAlign: 'left' }}>
-                      {invites.map(
-                        ([name, number]) => (
-                          <li key={name}>
-                            {String(number)} {name}
-                          </li>
-                        )
-                      )}
-                    </ul>
-                  ) : (
-                    'None'
-                  )}
-              </div>
-            )}
-          </div>
+            </div>
+          )}
         </div>
-        <button onClick={onClick}>{buttonLabel}</button>
+      </div>
+      <div style={{ textAlign: 'left' }}>
+        <h4  >GIVE </h4>
```

Comment:
Nit: remove empty space after `h4`

Looks like we have prettier set up already, but no `.workflow` in this repo to use it. Eventually we can add something like https://github.com/Agoric/dapp-psm/blob/main/.github/workflows/test.yml

Also, maybe something in the README to suggest the prettier extension in vscode to get auto-format-on-save